.github/workflows/zowe-explorer-ci.yml

```diff
@@ -1,9 +1,6 @@
 name: Zowe Explorer CI
 
-on:
-  push:
-    paths:
-    - packages/zowe-explorer/*
+on: [push, pull_request]
```

Comment:
should 
```
on:
  push:
    paths:
    - packages/zowe-explorer/**
```
be added back so that the Zowe Explorer CI workflow doesn't run if someone update the npm package or the ftp extension?

Reply:
ok. how do I add the pull_request? because this CI doesn't run during PR when I have that.

Reply:
plus if I have this path, it won't build correctly because we need files from zowe-explorer-api folder

Reply:
I think it will look something like this?
```
on:
  push:
    paths:
    - packages/zowe-explorer/**
  pull_request:
    paths:
    - packages/zowe-explorer/**
```

And about needing files from zowe-explorer-api, I believe that should be resolved by importing things from the npm package instead of local references?

Reply:
ok. i'll test it out. thank you

Reply:
What's weird is I can't see the Zowe Explorer CI in the checks